mlrun/utils/helpers.py

```diff
@@ -406,13 +407,18 @@ def _set_artifact_path(task):
     return conf
 
 
-def tag_image(base: str):
-    ver = config.images_tag or config.version
-    if ver and (
-        base == "mlrun/mlrun" or (base.startswith("mlrun/ml-") and ":" not in base)
-    ):
-        base += ":" + ver
-    return base
+def enrich_image(image: str):
```

Comment:
enrich_image_ref? enrich_image_url?

Reply:
what's wrong with just image ? 
it's image everywhere - functions specs, config, etc...

Reply:
because there is not class named `Image` and a docker image has many parts that this method directly deals with.
There isn't just "an image string" and a "string which is not an image"
There is a registry, a repo, an image name and a tag. And to understand what this does I think it requires naming the different parts in distinct names, as well as somehow referring to the full url/ref to avoid confusion.
This was also the evolution of naming in other repos we deal with. I'm not saying this should change because it's different there, but that it evolved there because it was needed and is correct as code piles on that deals with mutating parts of docker image(s) url(s)